### PR TITLE
Move pipeline manual to repo

### PR DESCRIPTION
**Pull request by nmdefries**
### Description
Migrating from [Gdrive](https://docs.google.com/document/d/171n814gtrt_hwSZY1ZiN-nYqo2u-nzzBLR6n0RKXeY8/edit), plus adding more detail about statistical review and documentation.

### CODE REVIEW

**Comment by nmdefries** [comment]
@dsweber2 feel free to do as light or thorough a review as you want (or make changes directly if you prefer). I will be doing followup PRs to flesh out some of the sections.

**Comment by dsweber2** [comment]
general suggestion: either do linebreaks at 80 characters, or put every sentence on a separate line (I generally prefer this strategy). This makes diffs easier to work with.

File: _template_python/INDICATOR_DEV_GUIDE.md
Comment (dsweber2) [inline _template_python/INDICATOR_DEV_GUIDE.md:372]: :star_struck:

File: _template_python/INDICATOR_DEV_GUIDE.md
Comment (dsweber2) [inline _template_python/INDICATOR_DEV_GUIDE.md:375]:
```suggestion
* Missingness characteristics, especially if the data is missing with a pattern (on weekends, specific states, etc)
```

File: _template_python/INDICATOR_DEV_GUIDE.md
Comment (dsweber2) [inline _template_python/INDICATOR_DEV_GUIDE.md:393]: > (you need to set acquisition job up)

Do we have docs on how to do this at all? I'm still hazy on it. I guess it's the link down in ### Staging

File: _template_python/INDICATOR_DEV_GUIDE.md
Comment (dsweber2) [inline _template_python/INDICATOR_DEV_GUIDE.md:425]:
```suggestion
1. the indicator run: https://cronicle-prod-01.delphi.cmu.edu/#Schedule?sub=edit_event&id=elr5clgy6rs

2. the acquisition run: https://cronicle-prod-01.delphi.cmu.edu/#Schedule?sub=edit_event&id=elr5ctl7art

See [@korlaxxalrok](https://www.github.com/korlaxxalrok) or [@minhkhul](https://www.github.com/minhkhul) for more information.
```

might want to use a real indicator fwiw

**Comment by dsweber2** [comment]
retag me when you're happy w/it (or just want another review).
fun fact: if you're writing a review and new commits come in, it just disappears your saved comments without deleting them. Hopefully I got the duplicates.

File: _template_python/INDICATOR_DEV_GUIDE.md
Comment (nmdefries) [reply _template_python/INDICATOR_DEV_GUIDE.md:393]: Minh [added deployment info](https://github.com/cmu-delphi/covidcast-indicators/pull/1986) that should cover that.

File: _template_python/INDICATOR_DEV_GUIDE.md
Comment (nmdefries) [reply _template_python/INDICATOR_DEV_GUIDE.md:184]: No, but CTIS (the FB survey) was in R and had many issues with memory and runtime. The backfill projection pipeline is in R but is not yet in production. That might be worth rewriting in Python eventually. It was in R because that's the language Jingjing was using for model development. I don't know how easy or hard it would be to get a Python model to perform exactly the same.

Anyway, do you think I don't need to mention R at all? Just leave it at "Indicators should be written in python..."?

File: _template_python/INDICATOR_DEV_GUIDE.md
Comment (nmdefries) [reply _template_python/INDICATOR_DEV_GUIDE.md:260]: Zips always start with a non-zero value, so we don't need to do any special handling.

File: _template_python/INDICATOR_DEV_GUIDE.md
Comment (nmdefries) [reply _template_python/INDICATOR_DEV_GUIDE.md:426]: Replacing this section with Minh's PR.

File: _template_python/INDICATOR_DEV_GUIDE.md
Comment (nmdefries) [reply _template_python/INDICATOR_DEV_GUIDE.md:314]: I'm trying to introduce these "tags" to describe how we've historically named signals, and also how we want to name signals in the future.

Example historical names:

JHU raw: confirmed_cumulative_prop
JHU smoothed: confirmed_7dav_cumulative_prop

Google raw: s01_raw_search
Google smoothed: s01_smoothed_search

Change smoothed: smoothed_adj_outpatient_covid

HHS raw: confirmed_admissions_covid_1d
HHS smoothed: confirmed_admissions_covid_1d_7dav

So "smoothed" shows up in various places. "7dav" has a similar meaning and in my mind should be in the same position. It also shows up in various positions in signal names but most consistently shows up at the end, as in the HHS smoothed signal.

Does that mean that we should say "smoothed" also comes at the end of the name, so that people use it like that in the future? I don't think any existing names put it at the end.

I haven't come up with an entire naming strategy, just some general tips. Do you have thoughts on naming format? Or should we leave it kind of open-ended?

File: _template_python/INDICATOR_DEV_GUIDE.md
Comment (dsweber2) [reply _template_python/INDICATOR_DEV_GUIDE.md:184]: Probably don't need to mention it given who's likely developing these pipelines in the future?

File: _template_python/INDICATOR_DEV_GUIDE.md
Comment (dsweber2) [reply _template_python/INDICATOR_DEV_GUIDE.md:314]: oh, I guess I just didn't do a thorough enough survey to notice it showed up in the middle sometimes.

I would change the wording to be prescriptive if that's what you're trying to communicate, and saying it should be at the end does sound like a good idea.

It may be worth mentioning if we're mirroring another processed dataset, that keeping their naming conventions is probably preferable.

**Review by dsweber2 — APPROVED** [review]
Gonna mark as approved, though Minh's docs should get merged before this gets merged of course.

**Pull request merged**
